src/main/java/com/hladchenko/springrestapi/excetion/UserNotFoundException.java

```diff
@@ -0,0 +1,4 @@
+package com.hladchenko.springrestapi.excetion;
+
+public class UserNotFoundException extends Exception {
```

Comment:
why have you decided to extend your custom exception from Exception, not RuntimeException?

Reply:
It was an Intellij Idea suggestion.